x-pack/legacy/plugins/monitoring/server/lib/cluster/get_clusters_summary.js

```diff
@@ -28,6 +30,17 @@ export function getClustersSummary(clusters, kibanaUuid, isCcrEnabled) {
 
     const clusterName = get(clusterSettings, 'cluster.metadata.display_name', cluster.cluster_name);
 
+    // check for any missing licenses
+    if (!license) {
+      const clusterId = cluster.name || clusterName || clusterUuid;
+      server.log(
+        ['error', LOGGING_TAG],
+        'Could not find license information for cluster = \'' + clusterId + '\'. ' +
```

Comment:
Noticed these two messages are slightly different: https://github.com/elastic/kibana/pull/40660/files#diff-9d5a4565871604f5de571b339e551c7bR16 was this intentional?

Also if not can we maybe reuse `(new MonitoringLicenseError(clusterId)).description` message? I know this will add i18n translation to the log, so not sure if that would be the best route

Reply:
Unintentional, nice find! I don't think we've historically localized log messages so unless anyone knows of any evidence to the contrary, I'm inclined to leave as is